iscript/src/iscript/mac.py

```diff
@@ -1179,7 +1210,7 @@ async def notarize_behavior(config, task):
         # Unlock keychain again in case it's locked since previous unlock
         await unlock_keychain(sign_config["signing_keychain"], sign_config["keychain_password"])
         await update_keychain_search_path(config, sign_config["signing_keychain"])
-        await create_pkg_files(config, sign_config, all_paths)
+        await create_pkg_files(config, sign_config, all_paths, requirements_plist_path)
```

Comment:
I tend to prefer explicit `requirements_plist_path=requirements_plist_path` for kwargs, but a) i'm not sure if that's pythonic and b) it looks like we've already gone this route elsewhere. Non-blocking :)